intel_owl/settings/security.py

```diff
@@ -15,7 +15,6 @@
 else:
     WEB_CLIENT_URL = f"http://{WEB_CLIENT_DOMAIN}"
 
-SESSION_COOKIE_SAMESITE = "Strict"
```

Comment:
@eshaan7 can we actually remove this without having any issue?

Reply:
I have sent a message on Slack regarding this as well

Reply:
Just adding another pointer :D